Formula/curl-openssl.rb

```diff
@@ -0,0 +1,52 @@
+class CurlOpenssl < Formula
+  desc "Get a file from an HTTP, HTTPS or FTP server"
+  homepage "https://curl.haxx.se/"
+  url "https://curl.haxx.se/download/curl-7.62.0.tar.bz2"
+  mirror "http://curl.mirror.anstey.ca/curl-7.62.0.tar.bz2"
+  sha256 "7802c54076500be500b171fde786258579d60547a3a35b8c5a23d8c88e8f9620"
+
+  keg_only :versioned_formula
+
+  depends_on "pkg-config" => :build
+  depends_on "brotli"
+  depends_on "nghttp2-libonly"
+  depends_on "openldap"
+  depends_on "openssl"
+
+  def install
+    # Allow to build on Lion, lowering from the upstream setting of 10.8
+    ENV.append_to_cflags "-mmacosx-version-min=10.7" if MacOS.version <= :lion
+
+    args = %W[
```

Comment:
Wouldn't we need to specifiy the ldap library here like `--with-ldap-lib=#{Formula["openldap"].opt_prefix}` or does it figure that out by itself ?

Reply:
pkg-config doing its job, it gets detected from there.